doc/user/content/ingest-data/postgres/self-hosted.md

```diff
@@ -479,12 +569,66 @@ start by selecting the relevant option.
 
 ### 3. Monitor the ingestion status
 
+{{< tabs >}}
+
+{{< tab "Console">}}
+
+From the [**Database object explorer**](/console/data/) in the [Materialize
+console], you can see the status of the source you created.
+
+Before it starts consuming the replication stream, Materialize takes a snapshot of the relevant tables in your publication. Until this snapshot is complete, Materialize won't have the same view of your data as your PostgreSQL database.
+
+![Image of the new source status:
+snapshotting](/images/console/console-create-new/postgresql/new-source-status-snapshotting.png
```

Comment:
Since it isn't possible to comment on the image: can we reduce this one to just include the snapshotting bar, to save some space? Also, the image doesn't seem to include the actual snapshotting progress modal.

Reply:
There's a modal for snapshotting? I only saw the little snapshotting label.

Reply:
updated where I cropped -- but didn't see the modal.

Reply:
Like so:

<img width="646" alt="Screenshot 2024-11-22 at 00 27 18" src="https://github.com/user-attachments/assets/3b149ef9-291c-418a-8eab-3931239e229c">

Reply:
Oh wow!  That did not show up on mine (albeit, my 2 tables had like 10 rows or whatnot) -- but looking at my screen recording did not show this.  Eh.

Reply:
Since I can't seem to get that to show up, I modified and overlayed that image onto the snapshotting screen.

Reply:
Hum, I'm sure we can get a proper screenshot (and maybe find a bug somewhere that prevents you from seeing it)! Are you trying it out with an upstream database that isn't empty? I can give it a whirl, too, if it just doesn't work for you.